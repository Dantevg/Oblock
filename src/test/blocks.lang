{
	a = {
		x = 10
		y = 20
		yield x + y
	}
	print a
}
