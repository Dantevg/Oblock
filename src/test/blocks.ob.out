{
  x = 10
  y = 20
}
10
